Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4218,10 +4209,10 @@ static char32_t characterForCharacterOffset(const CharacterOffset& characterOffs
     char32_t ch = 0;
     unsigned offset = characterOffset.startIndex + characterOffset.offset;
     if (offset < characterOffset.node->textContent().length()) {
-// FIXME: Remove IGNORE_CLANG_WARNINGS macros once one of <rdar://problem/58615489&58615391> is fixed.
-IGNORE_CLANG_WARNINGS_BEGIN("conditional-uninitialized")
+        // FIXME: Remove IGNORE_CLANG_WARNINGS macros once one of <rdar://problem/58615489&58615391> is fixed.
+        IGNORE_CLANG_WARNINGS_BEGIN("conditional-uninitialized")
         U16_NEXT(characterOffset.node->textContent(), offset, characterOffset.node->textContent().length(), ch);
-IGNORE_CLANG_WARNINGS_END
+        IGNORE_CLANG_WARNINGS_END
```

Comment:
I forgot to check if ignoring this warning is still needed before running `git-webkit pr`, so doing now but unfortunately `build-webkit` is now rebuilding the entire codebase again 🤷😠. So I will know in about 35 minutes? 😄

(If I find it's not needed, then I will amend to remove the `BEGIN`/`END` macros in the first place.)